### ISSUE TITLE
./configure options

### ISSUE THREAD

**Issue by mcast** [cooperative-computing-lab/cctools#701]
Hi,

I was trying to build just `parrot` and `irods` from http://ccl.cse.nd.edu/software/files/cctools-4.3.3-source.tar.gz and the "it has a ./configure script so I'll just guess how it works" approach, and trying to switch off as much as possible to make building simple.

I have a build now thanks, just writing here to let you know it wasn't as simple as it looked.

My first try suggests an error in the `./configure --help` output,
```
$ ./configure --help
Linux cgp-1-1-02 2.6.32-44-server #98-Ubuntu SMP Mon Sep 24 17:41:33 UTC 2012 x86_64 x86_64 x86_64 GNU/Linux
checking for package compatibility...
parrot IS supported on LINUX X86_64
resource_monitor IS supported on LINUX
Use: configure [options]
Where options are:
  --help
  --prefix             <path>
  --debug
  --strict
  --globus-flavor      <flavor>
  --xrootd-arch        <arch>
  --tcp-low-port       <port>
  --tcp-high-port      <port>
  --with-PACKAGE-path  <path>
  --without-system-SYSTEM

Where PACKAGE may be:
    readline
    fuse
    globus
    irods
    mysql
    perl
    python
    python3
    xrootd
    zlib
    cvmfs
    swig

And SYSTEM may be:
    sand
    allpairs
    wavefront
    makeflow
    ftp-lite
    chirp
    parrot
    resource_monitor
    resource_monitor_visualizer
    makeflow_linker
    doc
$ ./configure --with-irods-path /usr/local/iRODS --without-system-{sand,allpairs,wavefront,makeflow{,_linker},ftp-lite,chirp,resource_monitor{,_visualizer},doc} 
Linux cgp-1-1-02 2.6.32-44-server #98-Ubuntu SMP Mon Sep 24 17:41:33 UTC 2012 x86_64 x86_64 x86_64 GNU/Linux
checking for package compatibility...
parrot IS supported on LINUX X86_64
resource_monitor IS supported on LINUX
*** skipping system 'allpairs' because of dependencies
Unknown argument --without-system-makeflow_linker
```
I continued by removing entries from the `--without-system-FOO` until I finally got it to build with
```
./configure --prefix ~/build --with-irods-path /usr/local/iRODS --without-system-{sand,allpairs,wavefront,makeflow,resource_monitor{,_visualizer},doc}
```

**Comment by btovar** [comment]
Matthew,

We appreciate the report!

**Comment by btovar** [comment]
Fixed on 4.3.4. 

Master does not have this problem.

**Comment by btovar** [comment]
Matthew,
You can obtain the new version at: http://ccl.cse.nd.edu/software/files/cctools-4.3.4-source.tar.gz
